File: aplicacaomvc/Deloitte_NF.Web/Models/Cliente.cs

namespace Deloitte_NF.Models
{
    /// <summary>
    /// Classe Cliente
    /// </summary>
    public class Cliente
    {
        /// <summary>
        /// Obtém e informa o Sequencial do Cliente
        /// </summary>
        /// <value>Sequencial do Cliente</value>
        public int IdCliente { get; set; }

        /// <summary>
        /// Obtém e informa o Nome do Cliente
        /// </summary>
        /// <value>Nome do Cliente</value>
        public string NomeCliente { get; set; }

        /// <summary>
        /// Obtém e informa o CNPJ do Cliente
        /// </summary>
        /// <value>CNPJ do Cliente</value>
        public string CNPJCliente { get; set; }

        /// <summary>
        /// Obtém e informa o Endereço do Cliente
        /// </summary>
        /// <value>Endereço do Cliente</value>
        public string EnderecoCliente { get; set; }
    }
}

File: aplicacaowebforms/Negocio/Entidades/Cliente.vb
Public Class Cliente
    Inherits Generico

    Private _cpfCnpj As String
    Private _enderecoCliente As String

    Public Property CnpjCliente As String
        Get
            Return _cpfCnpj
        End Get
        Set(value As String)
            _cpfCnpj = value
        End Set
    End Property

    Public Property EnderecoCliente As String
        Get
            Return _enderecoCliente
        End Get
        Set(value As String)
            _enderecoCliente = value
        End Set
    End Property
End Class